.github/workflows/e2e.yml

```diff
@@ -118,7 +114,7 @@ jobs:
                   INTEGRATION_TEST_HOST: localhost
 
             - name: Print container logs
-              if: failure()
+              if: ${{failure() || cancelled()}}
```

Comment:
if the tests are taking really long to fail and you cancel them, it's nice to be able to see the container logs